iso_config.yaml

```diff
@@ -0,0 +1,37 @@
+log_file: "/var/tmp/host-os-iso-creator.log"
+mock_config: "./extras/centOS/7.2/mock/spin-iso-epel-7-ppc64le.cfg"
+rpm_dir: "./results"
+kickstart_file: "host-os.ks"
+hostos_package_group: "virtualization-hypervisor"
+package_group_list:
+    - "@base"
+    - "@core"
+    - "@debugging"
+    - "@desktop-debugging"
+    - "@dial-up"
+    - "@directory-client"
+    - "@fonts"
+    - "@gnome-apps"
+    - "@gnome-desktop"
+    - "@guest-desktop-agents"
+    - "@input-methods"
+    - "@internet-applications"
+    - "@internet-browser"
+    - "@java-platform"
+    - "@multimedia"
+    - "@network-file-system-client"
+    - "@performance"
+    - "@perl-runtime"
+    - "@print-client"
+    - "@ruby-runtime"
+    - "@virtualization-client"
+    - "@virtualization-hypervisor"
+    - "@virtualization-tools"
+    - "@web-server"
+    - "@x11"
+mock_spin_repo:
+    name: "hostos"
+    dir: "/spin-repo"
+distro_repo_url:
+    base: "http://mirror.aus.stglabs.ibm.com/centos-altarch/7/os/ppc64le/"
+    updates: "http://mirror.aus.stglabs.ibm.com/centos-altarch/7/updates/ppc64le/"
```

Comment:
This is an internal mirror.  What about http://mirror.centos.org/altarch/7/os/ppc64le/ for 'base' and http://mirror.centos.org/altarch/7/updates/ppc64le/ for 'updates'?